sbtc/src/webhooks.rs

```diff
@@ -31,7 +31,17 @@ use stacks_common::types::chainstate::BlockHeaderHash;
 use stacks_common::types::chainstate::BurnchainHeaderHash;
 use stacks_common::types::chainstate::StacksBlockId;
 
-use crate::error::Error;
+/// Webhooks error
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Can we rename this `WebhookError`?

Reply:
Actually, can we add these error variants to the crate error and feature gate them as well? Everything should work after importing the crate `Error`. @cylewitruk any opinions here?

Reply:
The proposed change won't force users to import more types, but instead force them to import a type from a different place. Which reminds me, we should probably move this enum to the `error` module.